src/components/Grid/Cell/GridCell.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import cx from 'classnames';
+
+type GridCellProps = {
+  children: React.ReactNode;
+  className?: string;
+  column: 1 | '1' | 2 | '2' | 3 | '3' | 'left' | 'content';
+  ofColumns?: 1 | '1' | 2 | '2' | 3 | '3';
+  row?: 'top' | 'bottom';
```

Comment:
@creido-welly would love to get your thoughts on whether you think these props make sense - in terms of naming and whether you think they're logical.

As we have two sets of names for `grid__cell`s (one using strings - e.g. content-top - the other using numbers - e.g. 3-of-3) I struggled to come up with props that allow for that.

I've also added comments explaining the two methods in the file.

Reply:
Let's go thru it together